test/UnitTests/GitHub.App/Models/AccountModelTests.cs

```diff
@@ -119,13 +119,26 @@ public static bool BitmapSourcesAreEqual(BitmapSource image1, BitmapSource image
 
         public static byte[] BitmapSourceToBytes(BitmapSource image)
         {
-            var encoder = new BmpBitmapEncoder();
-            encoder.Frames.Add(BitmapFrame.Create(image));
-            using (MemoryStream ms = new MemoryStream())
+            byte[] data = new byte[] { };
+            if (image != null)
             {
-                encoder.Save(ms);
-                return ms.ToArray();
+                try
+                {
+                    var encoder = new BmpBitmapEncoder();
+                    encoder.Frames.Add(BitmapFrame.Create(image));
+                    using (MemoryStream ms = new MemoryStream())
+                    {
+                        encoder.Save(ms);
+                        data = ms.ToArray();
+                    }
+                    return data;
+                }
+                catch (Exception ex)
```

Comment:
I'm guessing this will show a build warning? Replace with `catch { }`. Trying to keep the build clean. 😉